flocker/control/httpapi.py

```diff
@@ -868,6 +868,19 @@ def replace_configuration(self, applications, deployment):
         except ConfigurationError as e:
             raise make_bad_request(code=BAD_REQUEST, description=unicode(e))
 
+    @app.route("/state/node_eras", methods=['GET'])
+    @user_documentation(
+        u"""
+        List the currently known nodes and their eras.
+
+        A node's era can be looked up by reading
+        /proc/sys/kernel/random/boot_id. If possible use eras to identify
```

Comment:
Since we're going to encourage people to look up this value, should we provide an abstraction over `boot_id` so we can make changes without disrupting everyone, if necessary?